validator_client/src/http_api/tests.rs

```diff
@@ -954,6 +997,31 @@ async fn validator_graffiti() {
         .await;
 }
 
+#[tokio::test]
+async fn validator_graffiti_api() {
+    ApiTester::new()
+        .await
+        .create_hd_validators(HdValidatorScenario {
+            count: 2,
+            specify_mnemonic: false,
+            key_derivation_path_offset: 0,
+            disabled: vec![],
+        })
+        .await
+        .assert_enabled_validators_count(2)
+        .assert_validators_count(2)
+        .set_graffiti(0, "Mr F was here")
+        .await
+        .test_get_graffiti(0, "Mr F was here")
+        .await
+        .test_set_graffiti(0, "Uncle Bill was here")
+        .await
+        .test_get_graffiti(0, "Uncle Bill was here")
+        .await
+        .test_delete_graffiti(0)
+        .await;
+}
+
```

Comment:
I've just come across this comment:
https://github.com/sigp/lighthouse/blob/7d537214b7a337cc18440cc1827502219e988bee/validator_client/src/http_api/mod.rs#L1162-L1165

The test may seem a bit unnecessary at first, but I think it's a good to have to detect accidental breakage in the future. Please add the new routes to the `routes_with_invalid_auth` test here:

https://github.com/sigp/lighthouse/blob/c0b6b92f2704198a2301fdbda0ca08c29ce7e99b/validator_client/src/http_api/tests.rs#L672